vunit/project.py

```diff
@@ -518,6 +508,17 @@ def update(self, source_file):
         LOGGER.debug('Wrote %s content_hash=%s', source_file.name, new_content_hash)
 
 
+def _get_sorted_files(files, compile_order):
```

Comment:
See previous comments about merging `_get_compile_order`, `_get_affected_files` and `_get_sorted_files`.